packages/Webkul/Admin/src/Resources/views/notifications/index.blade.php

```diff
@@ -187,14 +159,39 @@ class="inline-flex gap-x-1 items-center justify-between w-full max-w-max ltr:ml-
                         if (url) {
                             axios.get(url)
                                 .then(response => {
-                                    this.notifications = [];
+                                    this.userNotifications = [];
     
-                                    this.notifications = response.data.search_results.data;
+                                    this.userNotifications = response.data.search_results.data;
     
                                     this.pagination = response.data.search_results;
                                 });
                         }
-                    }
+                    },
+
+                    timeAgo(timestamp) {
+                        const now = new Date();
+                        const past = new Date(timestamp);
+                        const diffInSeconds = Math.floor((now - past) / 1000);
+
+                        const intervals = {
+                            year: 365 * 24 * 60 * 60,
+                            month: 30 * 24 * 60 * 60,
+                            week: 7 * 24 * 60 * 60,
+                            day: 24 * 60 * 60,
+                            hour: 60 * 60,
+                            minute: 60,
+                            second: 1
+                        };
+
+                        for (const [unit, secondsInUnit] of Object.entries(intervals)) {
+                            const count = Math.floor(diffInSeconds / secondsInUnit);
+                            if (count > 0) {
+                                return count === 1 ? `${count} ${unit} ago` : `${count} ${unit}s ago`;
+                            }
+                        }
+
+                        return "just now";
```

Comment:
add translation for this

Reply:
Resolved:
```
$this->created_at->diffForHumans()
```

This method provides a human-readable, localized value based on the application's current locale settings